lib/workers/bin/run_single_worker.rb

```diff
@@ -55,7 +55,7 @@
 require File.expand_path("../../../config/environment", __dir__)
 
 worker_class = worker_class.constantize
-worker_class.before_fork
+worker_class.preload_for_worker_role if worker_class.respond_to?(:preload_for_worker_role)
```

Comment:
fixed, thanks @Fryguy

Reply:
This might soon not be needed since I think the only thing that actually makes use of this is the `UIWorker`, and that is for the `UIConstants`, if I am not mistaken.  Might be something else that I am missing though.

Reply:
Scratch that, guess we configure the secret token in that line...

https://github.com/ManageIQ/manageiq/blob/874cad6/app/models/mixins/miq_web_server_worker_mixin.rb#L25-L27

Nevermind... 😞

Reply:
Sorry... grepping and responding as I find new stuff... sorry for the spam, but found something else:

https://github.com/ManageIQ/manageiq/blob/874cad6/lib/vmdb/initializer.rb#L13

Does that mean we do this twice anyway?

Reply:
> Does that mean we do this twice anyway?

@NickLaMuro Good catch.  We gate that if you're a `Rails::Server`.  We can probably better consolidate these in followup PRs

Reply:
> We can probably better consolidate these in followup PRs

Yup, for sure, and wasn't trying to suggest we do that in this PR.  The `before_fork` stuff was just something I noticed when I original was getting the `run_single_worker` script going, so thought I would make a mention of it since you are working on it here.